tensorflow/core/kernels/boosted_trees/boosted_trees.proto

```diff
@@ -57,6 +58,18 @@ message BucketizedSplit {
   int32 right_id = 4;
 }
 
+message DenseSplit {
```

Comment:
can you leave a todo for me in this proto 
//TODO(nponomareva): move out of boosted_trees and rename to trees.proto